### ISSUE TITLE
Tiles missing randomly

### ISSUE THREAD

**Issue by DrMacabre** [Coyote-A/ultimate-upscale-for-automatic1111#14]
Hi,

im having this issue sometime where one or more tiles are generated empty. not always, and not always the same tiles

![image](https://user-images.githubusercontent.com/49474452/212253887-e6a6d74b-ae37-4a0b-942e-a5fc90f20409.png)

**Comment by Jopezzia** [comment]
Try launch webui with --no-half-vae arg and set preview to Approx NN in Settings - User interface tab
![image](https://user-images.githubusercontent.com/17219767/212267533-e5815717-eef7-400b-913d-f832f374ffa2.png)

**Comment by DrMacabre** [comment]
Thanks, approx NN is so bad compared to Full, i would recommend to stay away from it.

**Comment by Jopezzia** [comment]
I have only 6gb vram and with full preview have same black images\tiles (rare but annoying, with default img2img and txt2img too). So I decided to use Approx NN. Yes, the quality is worse, since it just pulls out a frame in poor quality, but you can at least output every step.

**Comment by DrMacabre** [comment]
Probably not related with memory since i have 24gb, i notice it's only doing this when using anything V3 VAE, with a different VAE, i don't get any black tiles

**Comment by AugmentedRealityCat** [comment]
Those are strange symptoms, Doctor. 
Is Anything V3 VAE what you were referring to when you were talking about "some exotic VAE" on Reddit ?

**Comment by DrMacabre** [comment]
nope, the exotic VAE is vae-ft-mse-840000-ema-pruned but this one doesn't do black tile at all. Only anything v3 does it.

**Comment by andreszs** [comment]
> Thanks, approx NN is so bad compared to Full, i would recommend to stay away from it.

I already had Aprox NN since it's the default mode, yet black boxes still appeared in the upscaled image or it was entirely black. I fixed this by using [these webui command line parameters](https://www.andreszsogon.com/fix-stable-difussion-automatic1111-webui-on-geforce-gtx-16xx/) for my GTX 1660 Super.
